src/test/java/io/github/externschool/planner/service/RoleServiceImplTest.java

```diff
@@ -0,0 +1,52 @@
+package io.github.externschool.planner.service;
+
+import io.github.externschool.planner.entity.Role;
+import io.github.externschool.planner.exceptions.RoleNotFoundException;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+public class RoleServiceImplTest {
+
+    @MockBean
+    private RoleServiceImpl roleService;
+
```

Comment:
[``A Mock object is something used for unit testing. If you have an object whose methods you want to test, and those methods depend on some other object, you create a mock of the dependency rather than an actual instance of that dependency. This allows you to test your object in isolation.`` ](https://stackoverflow.com/questions/2128148/what-are-mock-objects-in-java)
This class is RoleServiceImplTest, that means it's written to test  RoleService (btw RoleServiceTest sounds better). When you're going to test anything, you can mock whatever else, but NOT the object you're trying to test. There's no sense in such a test if you're mocking your results beforehand. Please, never mock the object of your test, mock dependencies which can affect the result. 
To mock means `make a replica or imitation of something.`